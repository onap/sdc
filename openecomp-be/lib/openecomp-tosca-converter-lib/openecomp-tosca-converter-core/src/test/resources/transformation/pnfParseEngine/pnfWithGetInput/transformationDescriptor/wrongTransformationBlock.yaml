- transformation-for: nodeTemplate
  name: PnfExtCp transformation
  description: converts any node template that has a type tosca.nodes.nfv.PnfExtCp
  query:
    type: tosca.nodes.nfv.PnfExtCp
  conversions:
    - query:
        properties:
          trunk_mode:
      to-name: trunk_mode_copy
      to-value:
        strategy: copy
- transformation-for: wrongBlock
  name: PnfExtCp transformation
  description: converts any node template that has a type tosca.nodes.nfv.PnfExtCp
  query:
    type: tosca.nodes.nfv.PnfExtCp
  conversions:
    - query:
        properties:
          role:
      to-name: role
      to-value:
        strategy: copy