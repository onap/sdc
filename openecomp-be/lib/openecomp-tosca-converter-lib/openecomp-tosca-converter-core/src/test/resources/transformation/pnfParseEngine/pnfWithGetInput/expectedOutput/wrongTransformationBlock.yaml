topology_template:
  node_templates:
    pnfExtCp_3:
      properties:
        trunk_mode_copy: false
    pnfExtCp_2:
      properties:
        trunk_mode_copy: false
    pnfExtCp_1:
      properties:
        trunk_mode_copy: false